File: Modul 4/C/PRAK401-2310817210014-DamarjatiSuryoLaksono.c
#include <stdio.h>

int main() {
    int araara;
    char symbol;
    int i = 1;

    scanf("%d %c", &araara, &symbol);

    while (i <= 50) {
        if (i % araara == 0) {
            printf("%c ", symbol);
        } else {
            printf("%d ", i);
        }
        i++;
    }

    return 0;
}

File: Semester-1/Pemrograman-1/Modul-4/Python/PRAK401-2310817210014-DamarjatiSuryoLaksono.py
araara, symbol = map(str, input("").split())
araara = int(araara)

i = 1

while i <= 50:
    if i % araara == 0:
        print(f"{symbol} ", end="")
    else:
        print(f"{i} ", end="")
    i += 1
